dogmode/README.md

```diff
@@ -0,0 +1,18 @@
+# Dogmode Example
+
+- [Introduction](#introduction)
+
+## Introduction
+
+The dog mode example will be migrated to this repository component-by-component. The description
+below represents the overall example when it is finished.
+
+The dog mode allows a car owner to keep their dog safe while they are away from the car. If the
+ambient temperature is high, multiple different applications will interact with each other to
+ensure that the temperature inside the car is at a safe level for the dog. This works as follows:
+first, the dog mode logic application detects whether a dog is present, either by automatically

```

Comment:
maybe make this a numbered list of steps?

Reply:
I think I will leave this as-is for now. Diagrams will be added and I think at that time I can list these steps in a way that corresponds to the diagrams. This is also a pretty much as-is copy of what is in the current dogmode README, and I imagine it will change a bit as we migrate over. Put the numbered list suggestion in this issue: https://github.com/eclipse-chariott/chariott-example-applications/issues/4